collector/Makefile

```diff
@@ -54,7 +19,7 @@ connscrape: build-connscrape-test
 		-v "$(BASE_PATH):$(SRC_MOUNT_DIR)" \
 		connscrape-test "$(SRC_MOUNT_DIR)/collector/connscrape-test/connscrape-test.sh"
 
-unittest: collector
+unittest:
```

Comment:
This target should still work, but you'll need to exec into the image and run cmake/make yourself, at that point you should be able to directly run ctest though.

Reply:
That does not seem convenient. I should be able to run `make unittest` and have it work as normal.

Reply:
Honestly, I would rather just remove the target altogether, I just thought leaving it could be nice.

IMO, running the unit tests should be done by exec'ing into the container from a terminal anyways, `cd cmake-build && ctest` is really not that long to type and, once you are in there, it shortens to `ctest` or up arrow + \<CR\>